src/connector/src/sink/big_query.rs

```diff
@@ -814,6 +814,7 @@ impl StorageWriterClient {
             offset: None,
             trace_id: Uuid::new_v4().hyphenated().to_string(),
             missing_value_interpretations: HashMap::default(),
+            default_missing_value_interpretation: MissingValueInterpretation::DefaultValue as i32,
```

Comment:
@xxhZs please check if the field's default behavior is expected

Reply:
checked, no problem